app/db/models.py

```diff
@@ -63,19 +89,87 @@ class Model(Base):
     model_url = Column(CONSTRAINT_STR, nullable=False, unique=True)
     created_at = Column(DateTime, nullable=False, default=datetime.datetime.now)
     version = Column(Integer, nullable=False, default=1)
+    expected_image_height = Column(Integer, nullable=False)
+    expected_image_width = Column(Integer, nullable=False)
+    type = Column(Enum(ModelType), nullable=False)
+    output_dtype = Column(Enum(*IMAGE_DTYPES, name="image_dtype"), nullable=False)
+    expected_bands = relationship(
```

Comment:
Can u explain this relation to the new "ModelBand" table to me? What kind of bands are we referring to here? All available bands offered by the image or just the bands that are used as inputs for the specific model? :)

Reply:
This is to specify the bands the model expects. This way the pipeline knows which bands to download from sentinelhub and send to the model

Reply:
Ahhhh, that makes sense, thx!